components/Cards/CreateCard.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { Paper, Box, Typography } from "@material-ui/core";
+import { useTheme } from "@material-ui/core/styles";
+import PhotoIcon from "@material-ui/icons/Photo";
+
+export default function CreateCard(props) {
+  const theme = useTheme();
+  const { spaceName, description } = props;
+  return (
+    <div>
+      <Box
+        style={{ background: theme.palette.primary.mainGradient }}
+        className="flex flex-grow justify-between rounded-r-xl rounded-bl-xl "
+      >
+        <div class="h-32 w-80 grid grid-cols-3">
+          <div class="p-6">
+            {/* spaceholder */}
+            <Box class="h-20 w-20 bg-purple-200"></Box>
+          </div>
+          <div class="col-span-2 pt-2">
+            <Box className="text-left p-3">
+              <Typography color="text" variant="h5" align="left">
+                {spaceName}
+              </Typography>
+              <Typography variant="body1" className="text-sm pt-1">
+                {description}
+              </Typography>
+            </Box>
+          </div>
+        </div>
+      </Box>
+    </div>
+  );
+}
```

Comment:
did eslint not give you an error for not having an extra line at the end? You turned it on right :3